### DISCUSSION TITLE
Is it possible to trigger an event to open/close all accordion's groups at once?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12066] [Archive]
With more than ten groups in the accordion, some users would like to

- see all the groups closed at a first sight: this allows them to see at once glance all the "sections" they will have to go through
- open them all, instead of clicking each time on the group's title. This makes it easier to go forth and back while feeding the form
- close them all for locating immediately the group they want to go to instead of scrolling a very long form.

Where can I find the code that opens or closes all accordion's groups?

**Comment by rijkvanzanten** [comment]
There's currently no event emitter in the app that would allow for this sort of operation

**Comment by Pachat** [comment]
Adding an event `onclick` on the accordion is what I have already done.
Now I just want to know which method to put into that will **Save and Stay** (because iI did not find in the source where `Ctrl+S` is captured).

**Reply by rijkvanzanten** [reply]
That shortcut is defined here:

https://github.com/directus/directus/blob/c1d705f94ad2061e6912d593a7183cb577cac6b2/app/src/modules/content/routes/item.vue#L347

The save and stay operation is defined here:

https://github.com/directus/directus/blob/c1d705f94ad2061e6912d593a7183cb577cac6b2/app/src/modules/content/routes/item.vue#L441-L456
